### PR TITLE
#165305767 Add Tags to Articles

### PR DESCRIPTION
**Pull request by Ogutu-Brian**
### What does this PR do?
- Adds Tags to articles
### What are the tasks to be completed?
* Ensure tags can be added during creating of article
* Add get all tags endpoint
* Ensure a user can update the tags on their articles
### How can this be manually tested?
a. **After cloning the repo:-**
```
$ git checkout  ft-tag-article-165305767
$ python manage.py runserver

```
b. **Test with postman**
1. Request
```
POST http://127.0.0.1:8000/api/articles/
```
2. Data
```
{
   "title": "How to train your dragon",
   "description": "Ever wonder how?",
   "body": "You have to believe",
   "tags":["names","Brian","test"]
}
```
3. Get request 
```
GET http://127.0.0.1:8000/api/articles/
```
4. Get all tags
```
http://127.0.0.1:8000/api/articles/tags/
```
*Screenshots*
image
image
image

### What are the relevant pivotal tracker stories?
[#165305767](https://www.pivotaltracker.com/story/show/165305767)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -389,12 +389,10 @@ Example request body:
 
 ```source-json
 {
-  "article": {
```
Comment (IssaIan) [inline README.md]: the article key was removed. Update the readme

**Review by matthenge — CHANGES_REQUESTED** [review]
@Ogutu-Brian  Please update your PR conversation since the example test data is not in that format, It throws an error in the way you have put it. Otherwise, everything seems to be in order. Great work!

**Review by IssaIan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Ogutu-Brian) [reply README.md]: I have done that @IssaIan

Comment (Ogutu-Brian) [reply README.md]: Hello @IssaIan ,
It is already updated..it has been deleted hence the pink colour

**Comment by Ogutu-Brian** [comment]
> @Ogutu-Brian Please update your PR conversation since the example test data is not in that format, It throws an error in the way you have put it. Otherwise, everything seems to be in order. Great work!

I have done that @matthenge

**Review by matthenge — APPROVED** [review]

**Review by matthenge — APPROVED** [review]
LGTM

**Pull request merged**